src/dailyLog/index.html

```diff
@@ -83,6 +79,8 @@ <h1>Date:</h1>
             </button>
             <!-- paragraph element to display the list of accomplishments -->
             <p class="accomplishment--list js-accomplishment-list"></p>
+            <!-- button to save log -->
+            <button type="button" id="save-button">Save Log</button>
```

Comment:
same here with adding a js-save-button class to select from in the JS rather than by id